psi4/src/psi4/libmints/pointgrp.cc

```diff
@@ -83,6 +83,7 @@
 #include "psi4/libpsi4util/PsiOutStream.h"
 #include "psi4/libpsi4util/exception.h"
 
+#include <bitset>
```

Comment:
Did you remove the `bitset` from the new function? If so, please remove the header.

Reply:
Good catch. Fixed.